packages/yarnpkg-pnpify/sources/sdks/cocvim.ts

```diff
@@ -0,0 +1,67 @@
+import {PortablePath, npath, ppath, xfs}                      from '@yarnpkg/fslib';
+import {PnpApi}                                               from '@yarnpkg/pnp';
+import CJSON                                                  from 'comment-json';
+import mergeWith                                              from 'lodash/mergeWith';
+
+import {Wrapper, GenerateIntegrationWrapper, IntegrationSdks} from '../generateSdk';
+
+export const merge = (object: unknown, source: unknown) =>
+  mergeWith(object, source, (objValue, srcValue) => {
+    if (Array.isArray(objValue))
+      return [...new Set(objValue.concat(srcValue))];
+
+    return undefined;
+  });
+
+export enum CocVimConfiguration {
+  settings = `coc-settings.json`,
+}
+
+export const addVSCodeWorkspaceConfiguration = async (pnpApi: PnpApi, type: CocVimConfiguration, patch: any) => {
```

Comment:
I think this should be `addCocVimWorkspaceConfiguration`. Maybe it would also be a good idea to move the `merge` and `addWorkspaceConfiguration` (an abstract implementation that every sdk file could extend with an `add<Editor>WorkspaceConfiguration` function that would call the base one) functions into a new `sdkUtils` file.